src/test/scala/QuartzSchedulerFunctionalSpec.scala

```diff
@@ -42,23 +45,45 @@ class QuartzSchedulerFunctionalSpec(_system: ActorSystem) extends TestKit(_syste
       }
 
     }
+
     "Properly Setup & Execute a Cron Job with correct fireTime" in {
       val receiver = _system.actorOf(Props(new ScheduleTestReceiver))
       val probe = TestProbe()
       receiver ! NewProbe(probe.ref)
       val jobDt = QuartzSchedulerExtension(_system).schedule("cronEvery10SecondsWithFireTime", receiver, MessageRequireFireTime(Tick))
 
-
       /* This is a somewhat questionable test as the timing between components may not match the tick off. */
       val receipt = probe.receiveWhile(Duration(1, MINUTES), Duration(15, SECONDS), 5) {
         case TockWithFireTime(scheduledFireTime) =>
           scheduledFireTime
       }
       0 until 5 foreach { i =>
-        assert(receipt(i)==jobDt.getTime+i*10*1000)
+        assert(receipt(i) === jobDt.getTime + i * 10 * 1000 +- tickTolerance)
```

Comment:
What's the reason for adding `tickTolerance` here? Was this test flaky?

Reply:
Exactly, the test was flakey.  Context: I forked this codebase and when running in my shared test/build these tests failed.